.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: required
+dist: trusty
+language: python
+python:
+  - "3.5"
+
+before_install:
+  - sudo apt-get update -q || true
```

Comment:
That config isn't automatically installing dependencies, not that that's a big deal. However, `clang-format-3.8 : Depends: libstdc++6 (>= 4.9) but 4.6.3-1ubuntu5 is to be installed` https://travis-ci.org/wpilibsuite/styleguide/jobs/160723947